### PR TITLE
Add debug information setup to capacitor for iOS

### PR DESCRIPTION
**Pull request by lucas-zimerman**
- This PR aims to instruct the user on how to add dsym symbols from iOS.
- Additionally, it adds a short description to the Apple session of uploading debug symbols.

solves the iOS doc on #4731

### CODE REVIEW

**Comment by bruno-garcia** [comment]
@imatwawana could you please let vercel publish the preview? Quite a few things moved around

**Review by imatwawana — COMMENTED** [review]
I left a comment about the front matter for a couple of the pages.

File: src/platforms/javascript/guides/capacitor/dsym.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+draft: false
+categories: []
+toc: true
+title: Uploading Debug Symbols (iOS)
+tags: []
```
[the remark this answers is not in the document]
Comment (lucas-zimerman) [reply src/platforms/javascript/guides/capacitor/dsym.mdx:6]: I agree. The steps from Sentry Capacitor and Apple should have a description set.

[the remark this answers is not in the document]
Comment (lucas-zimerman) [reply src/platforms/javascript/guides/capacitor/dsym.mdx]: @imatwawana Should I add the description to the Apple/iOS documentation on this PR or on a separated one?

[the remark this answers is not in the document]
Comment (imatwawana) [reply src/platforms/javascript/guides/capacitor/dsym.mdx]: You can do it on this one!

File: src/platforms/apple/common/dsym.mdx [hunk 2]
```diff
@@ -1,208 +1,7 @@
 ---
-draft: false
-categories: []
-toc: true
 title: Uploading Debug Symbols
-tags: []
+sidebar_order: 20
```
Comment (imatwawana) [inline src/platforms/apple/common/dsym.mdx:3]:
```suggestion
sidebar_order: 1
```

File: src/platforms/javascript/guides/capacitor/dsym.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Uploading Debug Symbols (iOS)
+sidebar_order: 20
```
Comment (imatwawana) [inline src/platforms/javascript/guides/capacitor/dsym.mdx:3]: I'm not sure where this should fall in the TOC order. Maybe @bruno-garcia can advise on that.

Comment (imatwawana) [inline src/platforms/javascript/guides/capacitor/dsym.mdx:3]: I'm not sure where this should be in the TOC order. [@bruno-garcia](https://github.com/bruno-garcia) can advise on that. For Apple, I updated the sidebar_order to make it match where it is currently.

**Review by imatwawana — APPROVED** [review]
Just one comment about the sidebar_order setting for Capacitor, but otherwise this looks good to me!

File: src/platforms/javascript/guides/capacitor/dsym.mdx (hunk 3 above)
Comment (bruno-garcia) [reply src/platforms/javascript/guides/capacitor/dsym.mdx:3]: Oh I see the preview was approved already

Comment (bruno-garcia) [reply src/platforms/javascript/guides/capacitor/dsym.mdx:3]: The order LGTM

**Comment by bruno-garcia** [comment]
@lucas-zimerman I see the Upload dSYM but not about Android? Without our plugin, the stacktraces won't make sense for native (NDK) crashes, or if the app uses proguard

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**